.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,46 @@
+name: Feature request
+description: Suggest a feature for one of our libraries.
+labels: [feature]
+body:
+- type: dropdown
+  attributes:
+    label: Core Library
+    multiple: true
+    options: 
+      - "@azure/msal or msal"
+      - "@azure/msal-browser"
+      - "@azure/msal-node"
+      - "passport-azure-ad"
+      - "adal-node"
+      - "adal.js"
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Wrapper Library
+    multiple: true
+    options:
+      - "@azure/msal-angular"
+      - "@azure/msal-react"
+      - "adal-angular"
+      - "Not Applicable"
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Description
+    description: "Please briefly describe the feature you are requesting."
+  validations: 
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Source
+    description: "Are you an internal or external customer?"
+    options: 
+      - Internal (Microsoft)
+      - External (Customer)
+  validations: 
+     required: true
```

Comment:
Not linked to this but related - we need to re-haul our roadmap structure for minor feature additions too, since some of them do not reflect in roadmap and we have multiple PRs open.